roles/letsencrypt/tasks/dns-challenge-os.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: add new TXT record to DNS zone
+  openstack.cloud.recordset:
+    auth:
+      auth_url: "{{ dns_os_auth_url }}"
+      username: "{{ dns_user }}"
+      password: "{{ dns_password }}"
+      project_name: "{{ dns_os_project_name }}"
+      user_domain_name: "{{ dns_os_user_domain }}"
+    state: present
+    recordset_type: "txt"
+    records:
+      - "\"{{ challenge['challenge_data'][item]['dns-01']['resource_value'] }}\""
```

Comment:
In case of txt-records the API request only works with doublequotes. (tested)
Reference: [https://docs.otc.t-systems.com/en-us/api/dns/dns_api_64001.html](https://docs.otc.t-systems.com/en-us/api/dns/dns_api_64001.html) ("Request" -> "Example request" -> "TXT type"